.travis.yml

```diff
@@ -2,19 +2,24 @@ language: python
 
 sudo: false
 
-addons:
-  apt:
-    sources:
-      - sourceline: deb https://packages.erlang-solutions.com/ubuntu trusty contrib
-        key_url: https://packages.erlang-solutions.com/ubuntu/erlang_solutions.asc
-    packages:
-      - erlang-nox
+services:
+  - rabbitmq
+
+# TODO:
+# set 'sudo: false' and uncomment once "Hash Sum mismatch" failure in 'apt-get update' is fixed
```

Comment:
Where is `apt-get update` called?

Reply:
We don't need it now as we're not installing anything with these changes and using the already-installed Erlang and RabbitMQ